pyproject.toml

```diff
@@ -41,6 +41,10 @@ test = [
     "pytest-asyncio >= 0.23.5",
 ]
 
+[tool.poetry.group.sanity.dependencies]
```

Comment:
If you use `uvx` in the Makefile, it will get and install ruff for you so that you don't need this group.

Reply:
ok